.travis/install.sh

```diff
@@ -15,3 +15,5 @@ pip install ${INSTALL_OPT} readme_renderer
 pip install ${INSTALL_OPT} pyinstaller
 pip install ${INSTALL_OPT} -r requirements-dev.txt
 pip install ${INSTALL_OPT} -e .
+
+pip list --outdated && true
```

Comment:
why do you have the `&& true` here? if you want make sure that the expression will always be true, you need to use a `|| true`

Reply:
oh, my bash skills 😞

Reply:
This actually doesn't return error codes.